x/auth/client/cli/tx_sign.go

```diff
@@ -230,64 +226,52 @@ func makeSignCmd() func(cmd *cobra.Command, args []string) error {
 			return err
 		}
 
-		// if --signature-only is on, then override --append
-		generateSignatureOnly, _ := cmd.Flags().GetBool(flagSigOnly)
-		multisigAddrStr, _ := cmd.Flags().GetString(flagMultisig)
-
-		from, _ := cmd.Flags().GetString(flags.FlagFrom)
+		printSignatureOnly, _ := f.GetBool(flagSigOnly)
+		multisigAddrStr, _ := f.GetString(flagMultisig)
+		from, _ := f.GetString(flags.FlagFrom)
 		_, fromName, err := client.GetFromFields(txFactory.Keybase(), from, clientCtx.GenerateOnly)
 		if err != nil {
 			return fmt.Errorf("error getting account from keybase: %w", err)
 		}
 
+		overwrite, _ := f.GetBool(flagOverwrite)
 		if multisigAddrStr != "" {
 			var multisigAddr sdk.AccAddress
-
 			multisigAddr, err = sdk.AccAddressFromBech32(multisigAddrStr)
 			if err != nil {
 				return err
 			}
-
 			err = authclient.SignTxWithSignerAddress(
-				txF, clientCtx, multisigAddr, fromName, txBuilder, clientCtx.Offline,
-			)
-			generateSignatureOnly = true
+				txF, clientCtx, multisigAddr, fromName, txBuilder, clientCtx.Offline, overwrite)
```

Comment:
`overwrite` should be ignored here, and should generate the full signature list for multisig transactions, in keeping with previous expected behavior. right?

Reply:
Default is `--overwrite=false`, which makes same result as before (`--append=true`), and same as we want it to be.

Reply:
As i mention above, sure. I don't mean to bike shed over this.

Reply:
> Default is --overwrite=false, which makes same result as before (--append=true)

True that. Maybe overwrite should be forced to true then when a `--multisig` address is passed. But I don't think it makes much difference 'cause the command should just print the signature in multisig mode, am I right?

Reply:
yes